src/claimtrie.cpp

```diff
@@ -282,29 +290,31 @@ bool CClaimTrie::haveClaimInQueue(const std::string& name, const COutPoint& outP
     {
         return false;
     }
+    const std::string normalized_name = lbryNormalize(name, CHECK_HEIGHT(nValidAtHeight));
```

Comment:
it looks like all the vars here are camelCase. should `normalized_name` be `normalizedName`?

Reply:
Sure, they can be.